README.md

```diff
@@ -0,0 +1,117 @@
+# specfile
+
+Python library for parsing and manipulating RPM spec files. Main focus is on modifying existing spec files, any change should result in a minimal diff.
+
+## Caveats
+
+### RPM macros
+
+specfile uses RPM for parsing spec files and macro expansion. Unfortunately, macros are always stored in a global context, which poses a problem for multiple instances of Specfile.
+
+## Examples and use cases
+
+### Instantiating
+
+```python
+# using an absolute path
+specfile = Specfile('/tmp/test.spec')
```

Comment:
Hm, I think we should unify the naming a bit. The introduction paragraph uses `spec file` (note the space), whereas the name of the class is `Specfile`. We should either use `RPM specfile` above, or name the class `SpecFile` in my opinion

Reply:
`RPM spec file` IMO equals to `RPM .spec file`, and it should stay that way, so it is clear even to those who have no idea what's going on. I thought about `SpecFile` at first, but then I decided to go with the single-word variant (which is being used quite often I think), because it's more straightforward, you can write `specfile` and not `spec_file` etc.

Also, packit uses Specfile as well:
https://github.com/packit/packit/blob/4f7a3690c68c1bf730678e92cc648d8610c850dd/packit/specfile.py#L25

Reply:
this is a good point

on the other hand, we started unifying on "spec file" in our user documentation

+1 Specfile class name